docs/user-guide/src/syntax.md

```diff
@@ -14,33 +14,78 @@ Prusti specifications are a superset of Rust boolean expressions. They must be d
 | [`exists(...)`](#quantifiers) | Existential quantifier |
 | [<code>... &#x7C;= ...</code>](#specification-entailments) | Specification entailment |
 
-## Old expressions
 
-Old expressions are used to refer to the value that a memory location pointed at by a mutable reference had at the beginning of the function:
+## `result` Variable
+
+When using Prusti, `result` is used to refer to what a function returns.
+`result` can only be used inside a postcondition, meaning that variables called `result` used in a function need to be renamed.
```

Comment:
Only arguments, local variables don't matter (you can't refer to them from specifications anyway).